_includes/footer.html

```diff
@@ -26,7 +26,7 @@ <h3 class="alt-h3 mb-3">Subscribe to updates</h3>
 
               <div class="mc-field-group col-12">
                   <label for="mce-EMAIL" class="d-none">Email Address</label>
-                  <input type="email" placeholder="Email Address" name="EMAIL" class="form-input required email d-block col-10 mx-auto py-2 px-3 mb-3" id="mce-EMAIL">
+                  <input type="email" placeholder="Email Address" name="EMAIL" class="form-input required email d-block col-10 mx-auto py-2 px-3 mb-3" id="mce-EMAIL" autocomplete="home email">
```

Comment:
Should the value not just be `email` instead of `home email` like it is now? As per [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) (<kbd>CTRL+F</kbd> for "autocomplete")